init.js

```diff
@@ -48,6 +58,85 @@ function reset() {
     oldTotals.parentNode.replaceChild(newTotals, oldTotals)
 }
 
+function loadItems(settings, dontChangeHash) {
+    if ("items" in settings && settings.items != "") {
+        // reset list of items
+        if (build_targets) {
+            build_targets = []
+            var targets = document.getElementById("targets")
+            while (targets.childNodes.length > 1) {
+                targets.removeChild(targets.firstChild);
+            }
+        }
+
+        var targets = settings.items.split(",")
+        for (var i = 0; i < targets.length; i++) {
+            var targetString = targets[i]
+            var parts = targetString.split(":")
+            var name = parts[0]
+            var target = addTarget(name)
+            var type = parts[1]
+            if (type == "f") {
+                target.setFactories(parts[2])
+            } else if (type == "r") {
+                target.setRate(parts[2])
+            } else {
+                throw new Error("unknown target type")
+            }
+        }
+    } else if (!build_targets.length) {
+        addTarget()
+    }
+    itemUpdate(dontChangeHash)
```

Comment:
This should not be called here. It should be called in loadData(), immediately following the loadItems() and loadModules() calls.